mindjunkies/courses/models.py

```
@@ -52,15 +58,16 @@ class Course(BaseModel):
         blank=True,
     )
 
-    published = models.BooleanField(default=False)
+    status = models.CharField(max_length=10, choices=STATUS_CHOICES, default="draft", blank=True)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)